### ISSUE TITLE
Nonfunctioning links on the Temp Outfitter's Permit application form - Chrome and Firefox

### ISSUE THREAD

**Issue by JohnHewittCriterion** [flexion/fs-intake-module#829]
Steps To Verify With JAWS:
 
Launch the website in Chrome or Firefox.
Activate the "Temp outfitter's permitLogin to apply now with login.gov logo".
Locate the list of new links, Steps 1 - 10.
Attempt to activate each of the links. No action is taken. Focus does not move.
Open the links list box with the key combination Insert + F7.
Press Enter on any of the "Step" links. Focus moves to the link on the page.

WCAG: 3.2 Predictable - 3.2.2 On Input 
EPERM-135

**Comment by sethalt** [comment]
This should be completed with the completion of 824

**Comment by sethalt** [comment]
- [ ] Check with Mac Voice Over
- [ ] Check with JAWS
